dotnet/agent-nr-config/README.md

```diff
@@ -0,0 +1,40 @@
+# OpenTelemetry Automatic Instrumentation (Agent) New Relic Config
+
+## Introduction
+
+This project demonstrates a simple .NET application running with the OpenTelemetry Automatic Instrumentation (Agent) configured to write data to New Relic. Docker is leveraged so that it is easier to demonstrate the following and make it easier to run the example:
+
+- Where to dowload the agent.
+- How to install the agent.
+- How to configure the agent.
+- How to build and run the application being monitored by the agent.
+
+The project consists of two main components:
+
+1. The default .NET Weather Forecast example application. This application does not directly contain any OpenTelemetry code or references.
+2. [Dockerfile](./Dockerfile): Contains the code to install, configure, and run the application while being monitored by the agent.
```

Comment:
I'm having a hard time with phrasing it succinctly. 
The dockerfile controls all of the following:
- build the example application
- download the agent
- install the agent
- configure the agent
- runs the example application with the agent enabled so that it can collect the telemetry data from the application

Reply:
How about something like:

>Contains the code to download, install, and configure the agent. Also builds and runs the example application with the agent enabled so that it can collect the telemetry data from the application.